config/webpack.test.js

```diff
@@ -204,7 +218,7 @@ module.exports = {
   tslint: {
     emitErrors: false,
     failOnHint: false,
-    resourcePath: 'src'
+    resourcePath: root
```

Comment:
I don't know what does root mean. Is it a keyword, which points to the root directory of the project? If it is so, then why are you linting over node_modules and stuff? Shouldn't we only run linting on the code we write?